client/landing/gutenboarding/stores/domain-suggestions/types.ts

```diff
@@ -1,9 +1,15 @@
-enum ActionType {
+/**
+ * External dependencies
+ */
+import { InputArgs } from '@wordpress/url';
+
+export enum ActionType {
```

Comment:
Oh, did we need this?

Reply:
This should probably be exported. Want to add it in #38401?

Reply:
It is being exported, just [on a separate line](https://github.com/Automattic/wp-calypso/pull/38219/files#diff-73c1a0ec8dafaa32d1056b6b2d10ca26L4). I can add the change to #38401 though.